test/webapi/ows/stac/test_controllers.py

```diff
@@ -148,6 +148,16 @@
 }
 
 
+def assert_equal_deep_dict(self, dict1, dict2):
```

Comment:
I add this function to test for nested dictionaries.

Reply:
Ok. However, you should keep track of the path and output it as part of the error message.

Reply:
But actually it is strange why this is needed. `TestCase.assertEquals()` actually does a deep equality comparison.

Reply:
`TestCase.assertEqual()` also checks for the order of the keys in the dictionary, which gives me an error.

Reply:
`TestCase.assertEqual()` also checks for the order of the keys in the dictionary, which gives me an error.

Reply:
> Ok. However, you should keep track of the path and output it as part of the error message.

done

Reply:
> But actually it is strange why this is needed. `TestCase.assertEquals()` actually does a deep equality comparison.

`TestCase.assertEqual()` also checks for the order of the keys in the dictionary, which gives me an error.